Assets/Code/Interactables/Effects/Static Effects/Model/StaticEffect.cs

```diff
@@ -0,0 +1,40 @@
+﻿public abstract class StaticEffect {
+    // StaticEffects are created BY DynamicEffects and persist across a specific timeframe (a turn, multiple turns, for the rest of the game, etc.)
+    // However, it is possible that a StaticEffect could, in turn, create DynamicEffects as well
+    // For example, a StaticEffect could be something such as a passive of drawing an extra card every turn
+    // The passive would CREATE a DrawCard DynamicEffect every turn
+    // StaticEffects can also be attached to specific entities in the game (status on a character, modifier on a value)
+    public int id;
+    public string name;
+
+    // There are primarily 2 different types of StaticEffects
+    // Modifiers, which directly influence the value of certain parameters, such as attack on a character
+    // Modifiers can only be attached to specific characters and entities, they cannot be passive
+    //
+    // Statuses, which tend to be more "special" effects that often require unique implementation
+    // Statuses can be both attached and passive
+    public enum ModifierType {
```

Comment:
should split these enums into their own files imo

Reply:
probably, yeah